extensions/product-recs/content/pom.xml

```diff
@@ -0,0 +1,242 @@
+    <parent>
+        <groupId>com.adobe.commerce.cif</groupId>
+        <artifactId>core-cif-components-parent</artifactId>
+        <version>1.11.1-SNAPSHOT</version>
+        <relativePath>../../../parent/pom.xml</relativePath>
+    </parent>
+
+    <!-- ====================================================================== -->
+    <!-- P R O J E C T  D E S C R I P T I O N                                   -->
+    <!-- ====================================================================== -->
+    <artifactId>core-cif-components-extensions-product-recs-content</artifactId>
+    <packaging>content-package</packaging>
+
+    <name>AEM CIF Core Components Products Recommendations Content Package</name>
+    <url>https://github.com/adobe/aem-core-cif-components</url>
+
+    <scm>
+        <connection>scm:git:https://github.com/adobe/aem-core-cif-components</connection>
+        <developerConnection>scm:git:git@github.com:adobe/aem-core-cif-components.git</developerConnection>
+        <url>https://github.com/adobe/aem-core-cif-components</url>
+        <tag>HEAD</tag>
+    </scm>
+    <organization>
+        <name>Adobe</name>
+        <url>https://www.adobe.com</url>
+    </organization>
+    <licenses>
+        <license>
+            <name>The Apache License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+        </license>
+    </licenses>
+
+    <!-- ====================================================================== -->
+    <!-- P R O P E R T I E S                                                    -->
+    <!-- ====================================================================== -->
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <vault.package.version>${project.version}</vault.package.version>
+    </properties>
+
+    <!-- ====================================================================== -->
+    <!-- B U I L D   D E F I N I T I O N                                        -->
+    <!-- ====================================================================== -->
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.jackrabbit</groupId>
+                <artifactId>filevault-package-maven-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <group>${vault.package.group}</group>
+                    <packageType>application</packageType>
+                    <properties>
+                        <acHandling>overwrite</acHandling>
+                        <description>${project.description}</description>
+                        <created>${buildNumber}</created>
+                        <createdBy>${vault.package.company}</createdBy>
+                        <cloudManagerTarget>none</cloudManagerTarget>
+                    </properties>
+                    <dependencies>
+                        <dependency>
```

Comment:
This should not be needed, since `https://github.com/adobe/aem-core-cif-components/blob/master/ui.apps/pom.xml` already has this dependency and this packages depends on `core-cif-components-apps`